### PR TITLE
[da]: Made a bunch of fixes to da locale

### PR DESCRIPTION
**Pull request by jezzdk**
I didn't go through all of them to be honest, just the ones I've come across and some I noticed while I was editing the file.

### CODE REVIEW

**Comment by andrey-helldar** [comment]
@jensstigaard, can you check this PR?

File: locales/da/json.json [hunk 1]
```diff
@@ -208,7 +208,7 @@
     "Continue To Payment": "Fortsæt til betaling",
     "Cook Islands": "Cookøerne",
     "Copy to clipboard": "Kopier til udklipsholder",
-    "Copy To Clipboard": "Kopier til udklipsholder",
+    "Copy To Clipboard": "Kopier til Udklipsholder",
```
Comment (jensstigaard) [inline locales/da/json.json:211]: "udklipsholder" should remain lower case.

File: locales/da/json.json [hunk 2]
```diff
@@ -260,22 +260,22 @@
...
     "Delete Token": "Slet token",
     "Deleting": "Sletter",
-    "Denmark": "Dansk",
-    "Detach": "Frigøre",
-    "Detach Resource": "Løsn Ressource",
-    "Detach Selected": "Løs Valgte",
-    "Details": "Information",
+    "Denmark": "Danmark",
+    "Detach": "Frigør",
+    "Detach Resource": "Frigør Ressource",
```
Comment (jensstigaard) [inline locales/da/json.json:265]: "Ressource" should be lower cased

File: locales/da/json.json [hunk 3]
```diff
@@ -260,22 +260,22 @@
...
+    "Detach Selected": "Frigør Valgte",
```
Comment (jensstigaard) [inline locales/da/json.json:266]: "Valgte" should be lower cased

File: locales/da/json.json [hunk 4]
```diff
@@ -260,22 +260,22 @@
...
+    "Details": "Informationer",
     "Developer": "Udvikler",
-    "Disable": "Deaktivere",
+    "Disable": "Deaktiver",
```
Comment (jensstigaard) [inline locales/da/json.json:269]: Should be "Deaktivér" (with accent over e)

File: locales/da/json.json [hunk 5]
```diff
@@ -284,23 +284,23 @@
     "E-Mail": "E-mail",
     "E-Mail Address": "Email adresse",
     "Ecuador": "Ecuador",
-    "Edit": "Redigere",
+    "Edit": "Rediger",
```
Comment (jensstigaard) [inline locales/da/json.json:287]: Should be "Redigér" (with accent over e)

File: locales/da/json.json [hunk 6]
```diff
@@ -284,23 +284,23 @@
...
-    "Editor users have the ability to read, create, and update.": "Editor brugere har mulighed for at læse, oprette og opdatere.",
+    "Editor users have the ability to read, create, and update.": "Redaktører har mulighed for at læse, oprette og opdatere.",
     "Egypt": "Egypten",
     "El Salvador": "Salvador",
-    "Email": "Mail",
-    "Email Address": "mailadresse",
-    "Email Addresses": "Email adresse",
-    "Email Password Reset Link": "Link Til Nulstilling Af E-Mail-Adgangskode",
+    "Email": "Email",
+    "Email Address": "Email adresse",
```
Comment (jensstigaard) [inline locales/da/json.json:297]: Should be one of either "E-mailadresse", "Email-adresse" or "Mail-adresse"

File: locales/da/json.json [hunk 7]
```diff
@@ -284,23 +284,23 @@
...
+    "Email Addresses": "Email adresser",
+    "Email Password Reset Link": "Email med Link Til Nulstilling Af Adgangskode",
     "Email Us": "Email os",
     "Emergency Token": "Nødtoken",
     "Employee": "Medarbejder",
-    "Enable": "Aktivere",
+    "Enable": "Aktiver",
```
Comment (jensstigaard) [inline locales/da/json.json:303]: Should be "Aktivér" (with accent over e)

File: locales/da/json.json [hunk 8]
```diff
@@ -458,16 +458,16 @@
     "Load :perPage More": "Belastning :perside mere",
     "Loading": "Indlæser",
     "Loading Notifications": "Indlæser notifikationer",
-    "Log in": "Login",
-    "Log In": "Log på",
+    "Log in": "Log ind",
+    "Log In": "Log Ind",
```
Comment (jensstigaard) [inline locales/da/json.json:462]: Should be "Log ind" (lowercased i)

File: locales/da/json.json [hunk 9]
```diff
@@ -458,16 +458,16 @@
...
     "Log Out": "Log Ud",
     "Log out": "Log ud",
-    "Log Out Other Browser Sessions": "Log Af Andre Bro Browsersessioner",
+    "Log Out Other Browser Sessions": "Log ud af andre browsersessioner",
     "Log Viewer": "Logfremviser",
-    "Login": "Log ind",
+    "Login": "Logind",
```
Comment (jensstigaard) [inline locales/da/json.json:467]: Should be "Log ind"

File: locales/da/json.json [hunk 10]
```diff
@@ -458,16 +458,16 @@
...
     "Login Via Emergency Token": "Log ind via nødtoken",
-    "Logout": "Log ud",
-    "Logout Other Browser Sessions": "Log ud andre browsersessioner",
+    "Logout": "Logud",
```
Comment (jensstigaard) [inline locales/da/json.json:469]: Should be "Log ud"

File: locales/da/json.json [hunk 11]
```diff
@@ -458,16 +458,16 @@
...
+    "Logout Other Browser Sessions": "Logud af andre browsersessioner",
```
Comment (jensstigaard) [inline locales/da/json.json:470]: Should be "Log ud af[...]"

File: locales/da/json.json [hunk 12]
```diff
@@ -560,8 +560,8 @@
     "Niger": "Niger",
     "Nigeria": "Nigeria",
     "Niue": "Niue",
-    "No": "Ingen",
-    "No :resource matched the given criteria.": "Nr. :resource matchede de givne kriterier.",
+    "No": "Nej",
+    "No :resource matched the given criteria.": "Inger :resource matchede de givne kriterier.",
```
Comment (jensstigaard) [inline locales/da/json.json:564]: Should be "Ingen [...]"

File: locales/da/json.json [hunk 13]
```diff
@@ -705,12 +705,12 @@
...
-    "Resources": "Midler",
-    "resources": "midler",
-    "Restore": "Gendanne",
+    "Resources": "Ressourcer",
+    "resources": "ressourcer",
+    "Restore": "Gendan",
     "Restore Resource": "Gendan Ressource",
-    "Restore Selected": "Gendan Valgt",
-    "results": "Resultat",
+    "Restore Selected": "Gendan Valgte",
```
Comment (jensstigaard) [inline locales/da/json.json:712]: "Valgte" should be lowercased

File: locales/da/json.json [hunk 14]
```diff
@@ -720,7 +720,7 @@
     "Rhode Island": "Rhode Island",
     "Role": "Rolle",
     "Romania": "Rumænien",
-    "Run Action": "Køre Handling",
+    "Run Action": "Kør Handling",
```
Comment (jensstigaard) [inline locales/da/json.json:723]: Should be "Udfør handling"

File: locales/da/json.json [hunk 15]
```diff
@@ -1013,7 +1013,7 @@
     "Uzbekistan": "Usbekistan",
     "Value": "Værdi",
     "Vanuatu": "Vanuatu formand",
-    "VAT ID": "moms ID",
+    "VAT ID": "Moms ID",
```
Comment (jensstigaard) [inline locales/da/json.json:1016]: Should be "Momsnummer"

**Review by jensstigaard — CHANGES_REQUESTED** [review]
I have made a review of the changes with comments for suggested adjustments to the danish translations. Overall the changes looks ok. The adjustments suggest are minor.

**Comment by andrey-helldar** [comment]
@jensstigaard, can you check this PR again?

**Comment by jensstigaard** [comment]
> @jensstigaard, can you check this PR again?

Yep!

I amde the decision to make a complete review of the danish translations. I have neglected this for a while. This turned out that a lot of the translations could be improved.
I forked the work of @jezzdk and corrected the reviewed translations, and pushed the corrected translations to my own fork, which now contains more correct translations for danish.
My fork can be found at:
https://github.com/jensstigaard/Laravel-lang/tree/jezzdk/main

I am not aware whats the easiest way to pull this into the main repo. My initial thought are:
 - Approve + accept this pull request
 - Open another pull request based on my fork
 - Approve this pull request

I hope it makes sense.
@andrey-helldar If you agree to this, I will open a pull request for my fork

**Comment by andrey-helldar** [comment]
@jensstigaard, yes, you can accept this PR and you can create a new one.

@jensstigaard, @jezzdk, Tak!

**Pull request merged**